proto/penumbra/penumbra/core/component/shielded_pool/v1/shielded_pool.proto

```diff
@@ -60,6 +60,8 @@ message NotePayload {
   // An encryption of the newly created note.
   // 176 = 80(address) + 16(amount) + 32(asset ID) + 32(rseed) + 16(MAC) bytes.
   NoteCiphertext encrypted_note = 3;
+  // Whether to exclude this spend from the CompactBlock.
+  bool sync_exclude = 4;
```

Comment:
This shouldn't be a property of the `NotePayload`, which represents an encrypted note to be scanned, because the point of the field is to mark it as not-to-be-scanned. Instead, the field should go on the `OutputBody` proto message.